ui/main/src/app/modules/card/components/card-ack/card-ack.component.ts

```diff
@@ -191,14 +202,22 @@ export class CardAckComponent implements OnInit, OnChanges, OnDestroy {
                 this.card = {...this.card, hasBeenAcknowledged: false};
                 this.setAcknowledgeButtonVisibility();
                 OpfabStore.getLightCardStore().setLightCardAcknowledgment(this.card.id, false);
+
+                const childCards = OpfabStore.getLightCardStore().getChildCards(this.card.id);
+                if (childCards) {
+                    childCards.forEach(child => {
+                        if (child.actions?.includes(CardAction.PROPAGATE_READ_ACK_TO_PARENT_CARD)) {
+                            AcknowledgeService.deleteUserAcknowledgement(child.uid).subscribe();
+                        }
+                    })
+                }
```

Comment:
I think it is not necessary to cancel ack for child cards as if the parent ack is cancelled the card will be considered as unack whatever the status of the childs